src/libclient/CMakeLists.txt

```diff
@@ -0,0 +1,175 @@
+project(libowncloudclient)
```

Comment:
I agree with @sebasje, `libsyncui` is cool. We really want to avoid the word "owncloud" here.